web/api/v1/pod.ts

```diff
@@ -4,6 +4,26 @@ import { AxiosInstance } from "axios";
 
 export default function podAPI(k8sInstance: AxiosInstance) {
   return {
+    createPod: async (req: V1Pod) => {
+      try {
+        if (!req.metadata?.generateName) {
+          throw new Error(`metadata.generateName is not provided`);
+        }
+        req.kind = "Pod";
+        req.apiVersion = "v1";
+        if (req.metadata.managedFields) {
+          delete req.metadata.managedFields;
+        }
+        const res = await k8sInstance.post<V1Pod>(
+          namespaceURL + `/pods?fieldManager=simulator&force=true`,
```

Comment:
```suggestion
          namespaceURL + "/pods?fieldManager=simulator&force=true",
```
Essentially, I think `"` should be used in other existing string values...